emukit/model_wrappers/gpy_quadrature_wrappers.py

```diff
@@ -162,26 +164,57 @@ def dKdiag_dx(self, x: np.ndarray) -> np.ndarray:
         return np.zeros((input_dim, num_points))
 
 
-def convert_gpy_model_to_emukit_model(gpy_model, integral_bounds: List[Tuple[float, float]], integral_name: str = '') \
+def convert_gpy_model_to_emukit_model(gpy_model, integral_bounds: Union[None, List[Tuple[float, float]]],
+                                      measure: Union[None, IntegrationMeasure], integral_name: str = '') \
         -> BaseGaussianProcessGPy:
     """
-    Wraps a GPy model and returns an emukit quadrature model
+    Wraps a GPy model and returns an emukit quadrature model.
 
     :param gpy_model: A GPy Gaussian process regression model, GPy.models.GPRegression
     :param integral_bounds: List of D tuples, where D is the dimensionality of the integral and the tuples contain the
-        lower and upper bounds of the integral i.e., [(lb_1, ub_1), (lb_2, ub_2), ..., (lb_D, ub_D)]
+    lower and upper bounds of the integral i.e., [(lb_1, ub_1), (lb_2, ub_2), ..., (lb_D, ub_D)]. None means infinite
+    bounds.
+    :param measure: an integration measure. None means no measure.
     :param integral_name: the (variable) name(s) of the integral
 
-    :return: emukit model for quadrature witg GPy backend (IBaseGaussianProcessGPy)
+    :return: emukit model for quadrature with GPy backend (IBaseGaussianProcessGPy)
     """
 
-    # wrap standard kernel and get quadrature kernel
-    if gpy_model.kern.name is 'rbf':
+    # wrap standard kernel
+    if gpy_model.kern.name == 'rbf':
```

Comment:
is `ininstace` possible here?

Reply:
done. Also added an ARD check as we need ARD=False.